### DISCUSSION TITLE
Browser scale and mater detail

### DISCUSSION THREAD

**Discussion by Zahar661** [serenity-is/Serenity#6087] [Help]
Hello everyone! who knows how to defeat the master detail bug? at 90, 70, 50% records are not displayed until the window is resized.
90%
![image](https://user-images.githubusercontent.com/26430136/141929549-56c0dc9c-af2b-4ffd-a7a8-3ba2144d1d82.png)
80%
![image](https://user-images.githubusercontent.com/26430136/141929838-95d55240-419c-448f-8e1c-b5baf1f4b2f5.png)

**Comment by VictorTomaili** [comment]
I can't reproduce that, probably browser differences. In javascript, there is no event for zoom. And the `slickgrid` is a calculated virtual grid. This means it calculates the row heights and visible areas while scrolling, resizing, etc. Zooming is out of scope.

**Comment by Zahar661** [comment]
https://serenity.is/demo/Northwind/Customer#edit/1
Here, refresh the page from 90%. On the orders tab.

**Reply by VictorTomaili** [reply]
As I said, browser differences. 
![image](https://user-images.githubusercontent.com/2023440/142196300-3d8d7812-f7d6-41f9-aec7-bcc2a28b20ca.png)

For solving your problem, you can add a global interval for checking screen width and height, and refresh the grids on the screen if changed. 

![image](https://user-images.githubusercontent.com/2023440/142197145-0bf74914-42b2-411d-8605-2e73737618eb.png)

**Comment by Zahar661** [comment]
Changing global interval doesn't help ((  In all browsers on the chromium engine, this error appears.

**Reply by axelcasauran** [reply]
have you solve this issue?
